onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -484,6 +484,28 @@ def aten_arange_start(
     return result
 
 
+@torch_op("aten::arange.start_step", private=True)
+def _aten_arange_start_step_onnx(
+    start: TRealUnlessFloat16OrInt8,
+    end: TRealUnlessFloat16OrInt8,
+    step: TRealUnlessFloat16OrInt8,
+    dtype: int,
+) -> Tuple[FLOAT, FLOAT, FLOAT]:
```

Comment:
Note: comment that links to the pytorch ref implementation that explains this logic